plugins/admin.sma

```diff
@@ -444,7 +444,7 @@ public adminSql()
 
 		query = SQL_PrepareQuery(sql, "SELECT auth, password, access, flags FROM %s", table)
 	} else {
-		SQL_QueryAndIgnore(sql, "CREATE TABLE IF NOT EXISTS `%s` ( `auth` VARCHAR( 32 ) NOT NULL, `password` VARCHAR( 32 ) NOT NULL, `access` VARCHAR( 32 ) NOT NULL, `flags` VARCHAR( 32 ) NOT NULL ) COMMENT = 'AMX Mod X Admins'", table)
+		SQL_QueryAndIgnore(sql, "CREATE TABLE IF NOT EXISTS `%s` ( `auth` VARCHAR( 43 ) NOT NULL, `password` VARCHAR( 31 ) NOT NULL, `access` VARCHAR( 31 ) NOT NULL, `flags` VARCHAR( 31 ) NOT NULL ) COMMENT = 'AMX Mod X Admins'", table)
```

Comment:
No, it's because of 

https://github.com/alliedmodders/amxmodx/blob/6119ef7ae2ac461f22d9213910f71610fb923e0c/amxmodx/CMisc.h#L254

Reply:
The column sizes don't have anything to do with the C++ types, it shouldn't be necessary to change them.

Reply:
Of course it's related to the C++ types. The password column was changed from 32 to 31 because it would have been silently truncated before. The auth column from 32 to 43 to allow you to store as much as you would be able in a .ini or elsewhere.

If you mean that it's not related to the buffer overrun, that's fixed with the CMisc.h change. The sma changes are additional changes.